src/ontology/edits/oeo-physical.omn

```diff
@@ -6970,6 +6970,331 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1575",
         OEO_00240003
     
     
+Class: OEO_00010423
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A power generation technology is an energy technology that describes how to combine power generating units and energy carriers in a power generation process to generate electrical energy.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1572
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1601",
+        rdfs:label "power generation technology"@en
+    
+    SubClassOf: 
+        OEO_00020267,
+        <http://purl.obolibrary.org/obo/IAO_0000136> some 
+            (OEO_00000334
+             and (<http://purl.obolibrary.org/obo/RO_0000056> some OEO_00240014))
+    
+    
+Class: OEO_00010424
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A wind power technology is a power generation technology that describes how to combine wind energy converting units and a wind energy transformation.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1572
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1601",
+        rdfs:label "wind power technology"@en
+    
+    SubClassOf: 
+        OEO_00010423,
+        (<http://purl.obolibrary.org/obo/IAO_0000136> some OEO_00000044)
+         and (<http://purl.obolibrary.org/obo/RO_0000056> some OEO_00020043)
+    
+    
+Class: OEO_00010425
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "An onshore wind power technology is a wind power generation technology that describes how to use wind energy converting units in onshore wind farms.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1572
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1601",
+        rdfs:label "onshore wind power technology"@en
+    
+    SubClassOf: 
+        OEO_00010424,
+        (<http://purl.obolibrary.org/obo/IAO_0000136> some 
+            (OEO_00000044
+             and (<http://purl.obolibrary.org/obo/BFO_0000050> some OEO_00000311)))
+         and (<http://purl.obolibrary.org/obo/RO_0000056> some OEO_00020043)
+    
+    
+Class: OEO_00010426
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "An offshore wind power technology is a wind power generation technology that describes how to use wind energy converting units in offshore wind farms.",
```

Comment:
Is there a difference between:

A) ... that describes how to combine ...
B) ... that describes how to use ...

all other techs in the PR use A), only offshore&onshore wind use B)

Reply:
In this case, B sound more sensible. The "combines" is mentioned in the parent class already and thus holds for the subclasses, too.